projects/frontend/cicd/.gitlab-ci.yml

```diff
@@ -38,7 +38,8 @@ frontend-data-pipelines-build:
   before_script:
     - cd projects/frontend/
   script:
-    - ./cicd/install_data_pipelines.sh
+    - export SHARED_RC_VERSION=$(../../cicd/extract_shared_rc_version.sh $CI_PIPELINE_ID)
```

Comment:
what happens is shared didn't run in this pipeline? 
What value will it use for the shared rc version?

Reply:
It will just pass blank. If the pipeline hasn't run for shared, then there will be nothing in the registry and we'll get nothing when we pull the versions and filter. The install script will then build data-pipelines with the latest version. I'll test again just in case.

Reply:
can you document that here.

Reply:
Also worth noting: we do something similar in the increment script when we extract the version and increment it. If it's an empty string, it will still increment it and return 1, because bash considers empty string equivalent to zero when doing math.

Reply:
Documented it in both places